jujugui/static/gui/src/app/init/test-zip-utils.js

```diff
@@ -0,0 +1,279 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+'use strict';
+
+const zipUtils = require('./zip-utils');
+
+// XXX: disabled until we figure out how to mock required modules.
+xdescribe('Zip utils', function() {
```

Comment:
Can you create an issue to track this so it doesn't get lost.

Reply:
#3248